spec/googleauth/credentials_spec.rb

```diff
@@ -13,7 +13,8 @@
 # limitations under the License.
 
 require "googleauth"
-
+require "spec_helper"
+require "tmpdir"
```

Comment:
I'd expect to see some tests for the `Credentials#duplicate` functionality, that might for example have caught the `@client.logger` issue noted above.

Reply:
Added